doc/source/configuration.rst

```diff
@@ -43,6 +43,35 @@ A ``Rule`` is a dictionary with the following optional keys:
      - A `label <https://help.github.com/articles/about-labels/>`_ name that
        will disable Mergify if it is applied to a pull request. Default is
        ``no-mergify``.
+   * - ``merge_strategy``
+     - :ref:`Merge Strategy`
+     - The method to use to merge pull requests
+
+
+.. _merge_strategy:
+
+Merge Strategy
+==============
+
+A ``Merge Strategy`` defines the method to merge pull requests. It is a
+dictionary with the following optional keys:
+
+.. list-table::
+   :header-rows: 1
+
+
+   * - Key name
+     - Value type
+     - Value description
+   * - ``method``
+     - ``string``
+     - Merge method to use. Possible values are ``merge``, ``squash`` or
```

Comment:
> The method to use to merge the pull request.

Is there any good reason to default to `rebase` where I think GitHub defaults to `merge`?

Reply:
Because this is what we use in Gnocchi. rebase make more sense with "strict: True". We can switch back to Github default and update Gnocchi configuration.

Reply:
Yeah I'd say fix Gnocchi. It's confusing to change the GitHub default IMHO.